_plugins/editable_docs.rb

```diff
@@ -7,17 +7,16 @@
     end
     next unless page_path.start_with?("doc")
     repo_name, repo_data = repositories.find do |name, data|
-      page_path.start_with? (File.join("doc", name))
+      page_path.start_with?(File.join("doc", name))
     end
     next if repo_name.nil? or repo_data.nil?
-
     # Generate edit url
-    page_relative_url = page_path.sub(File.join("doc", repo_name), "")
-    page.data["edit_url"] = generate_edit_url(repo_data, page_relative_url)
+    original_filepath = page.data["file_relpath"].split('/')[2..-1].join('/')
```

Comment:
@apojomovsky why the new slicing here?

Reply:
This takes a path that is relative to the project's root and "goes up" two levels 
`_remotes/ros2_overview/Build-Cop-and-Build-Farmer-Guide.md` --> `Build-Cop-and-Build-Farmer-Guide.md`.
But I just revisited this code to make it more robust by using a sub-string search.